src/index.ejs

```diff
@@ -18,6 +17,7 @@
     <% } %>
   </head>
   <body>
+    <div id="init-drag-region" style="position: absolute;-webkit-app-region: drag;left: 0;right: 0;top: 0;height: 25px;"></div>
```

Comment:
What does this `init-drag-region` div use for? pls write a comment above this element, so that we know what does this element use for in the future.